README.md

```diff
@@ -18,3 +18,11 @@ Here's some of the other things we are using:
 * [Babel](https://babeljs.io/) - A transpiler for ES6
 * [Autoprefixer](https://github.com/middleman/middleman-autoprefixer) -
 Autoprefix CSS
+
+## Deploy
+
+Just run the deploy script:
```

Comment:
As per [this](http://www.tameri.com/edit/words2kill.html), let's try to avoid using "just". Ok?